### PR TITLE
V0 Skirts Dual color

### PR DESCRIPTION
**Pull request by doubletrouble023**
* [x ] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [ x] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [ x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [ x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x ] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [ x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [ x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [ x] Submitted STL files are oriented and scaled properly for printing.
  * [ x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?
Taller dual color skirts for the V0
#### How was it tested? 
Printed and installed 
#### Any background context you want to provide?
no
#### Screenshots (if appropriate)
no
#### Further notes
no

### CODE REVIEW

File: printer_mods/doubletrouble/README.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+V0 Skirt mod
+One piece, dual color, built in V0 Display and C14 in the rear
```
Comment (KiloQubit) [inline printer_mods/doubletrouble/README.md:2]: You have 3 images with your mod, can you include them in this README?

    Skirt 1
    Skirt 2
    Skirt 3

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for your PR!  I have noted some needed changes below.

- Please add a link to your mod in `printer_mods/README.md` in alphanumeric order.

- Please rename `w_SKiRT_ORANGE_sida_2x.stl` to `[a]_w_SKiRT_side_x2.stl` (side is misspelled as sida)

- Please change `2x` to `x2` in the filenames that need two copies to follow the same convention as used for other Voron parts.

- For accent parts, please name them starting with `[a]_` instead of putting `ORANGE` in them, to follow the naming convention of other Voron parts.

- Please break up `Tall_Feet_w.stl` into separate STLs - one part per STL.  This allows users to arrange the parts on the plate as they see fit.

- Please break up `w_SKiRT_ORANGE_Front.stl` into separate STLs - one part per STL.  This allows users to arrange the parts on the plate as they see fit.

- There's something going on inside `w_SKiRT_ORANGE_sida_2x.stl` and `w_SKiRT_ORANGE_C14.stl`, seems to be some kind of internal artifact that should not be there:

Screen Shot 2021-03-22 at 20 52 58

File: printer_mods/doubletrouble/README.md (hunk 1 above)
Comment (doubletrouble023) [reply printer_mods/doubletrouble/README.md]: Think everything should be fixed now

File: printer_mods/README.md [hunk 2]
```diff
@@ -33,6 +33,7 @@ like so:
 |Dan88 |[V0 one-part Skirts](./Dan88/V0skirt) |One part Skirts for the V0 |:heavy_check_mark: |:x: |:x: |
 |DorjeDorf | [Duet 3 6HC Din Bracket](./DorjeDorf/Duet_3_6HC_Din_Bracket) | V2.4 Duet 3 6HC Din Bracket | :x: | :x: | :heavy_check_mark: | 
 | |[Duet 3 3HC Din Bracket](./DorjeDorf/Duet_3_3HC_Din_Bracket) |  V2.4 Duet 3 3HC Din Bracket | :x: | :x: | :heavy_check_mark: | 
+|doubletrouble |[V0 Skirt Mod](./doubletrouble023/V0_Skirt_Mod) |V0 Skirt Mod |:x: |:x: |:heavy_check_mark: |
```
Comment (KiloQubit) [inline printer_mods/README.md:36]: Link is broken, the directory you used is `doubletrouble`, not `doubletrouble023`.  Also, you need a checkmark under the V0 column, not the V2.4 column.

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for the updates.  For some reason the accent front right skirt has a double underscore in the name between front and right - can you rename it to use a single underscore?  `[a]_w_Skirt_Front__right.stl` to `[a]_w_Skirt_Front_right.stl`

There's also an issue with the README, see below.

File: printer_mods/README.md (hunk 2 above)
Comment (doubletrouble023) [reply printer_mods/README.md]: Oops, sorry. Fixed now

**Review by KiloQubit — APPROVED** [review]
Thanks, looks good now!

**Comment by KiloQubit** [comment]
Now that this is merged, I recommend deleting your fork of VoronUsers because it's well out of date, and explaining how to keep it up to date with upstream is beyond the scope of this little comment box.  Fork it again right before the next time you want to make a pull request.

**Comment by doubletrouble023** [comment]
Thanks, think I've managed to delete it now. Probably won't try to add
anything more, will share it in other ways if necessary. Github is waaaaay
too complicated for this use. There must be a better platform we can use

**Comment by xbavnu** [comment]
Could you also provide the step-files? I want to make some minor changes to the models, but working with stl-files is a nightmare.

**Pull request merged**